react/features/always-on-top/AlwaysOnTop.js

```diff
@@ -203,6 +207,8 @@ export default class AlwaysOnTop extends Component<*, State> {
         window.addEventListener('mousemove', this._mouseMove);
 
         this._hideToolbarAfterTimeout();
+
+        this._defaultAvatar = api.getDefaultAvatarURL();
```

Comment:
I believe this._defaultAvatar should be set in the constructor so it's available on the first render.